src/Controller/OmniaController.php

```diff
@@ -427,72 +495,17 @@ protected function mapData(MediaInterface $media, $videoData) {
     }
   }
 
-  /**
-   * Map multiple omnia term ids to drupal term ids.
-   *
-   * @param int[] $omnia_ids
-   *   Array of omnia termn ids.
-   * @param int $vid
-   *   The drupal vid of the term.
-   *
-   * @return int[]
-   *   Array of mapped drupal ids, might contain less ids then the input array.
-   */
-  protected function mapMultipleTermIds(array $omnia_ids, $vid) {
-    $drupal_ids = [];
-    foreach ($omnia_ids as $omnia_id) {
-      $drupalId = $this->mapTermId($omnia_id, $vid);
-      if ($drupalId) {
-        $drupal_ids[] = $drupalId;
-      }
-      else {
-        $this->logger->warning('Unknown omnia ID @term_id"',
-          [
-            '@term_id' => $omnia_id,
-          ]
-        );
-      }
-    }
-
-    return $drupal_ids;
-  }
-
-  /**
-   * Map omnia term Id to corresponding drupal term id.
-   *
-   * @param int $omnia_id
-   *   The omnia id of the term.
-   * @param int $vid
-   *   The drupal vid of the term.
-   *
-   * @return int
-   *   The drupal id of the term.
-   */
-  protected function mapTermId($omnia_id, $vid) {
-    $result = $this->database->select('nexx_taxonomy_term_data', 'n')
-      ->fields('n', ['tid'])
-      ->condition('n.nexx_item_id', $omnia_id)
-      ->condition('n.vid', $vid)
-      ->execute();
-
-    $drupal_id = $result->fetchField();
-
-    return $drupal_id;
-  }
-
```

Comment:
Why did you move around those methods? It makes the PR unnecessary hard to read.